### ISSUE TITLE
Cant write on General/LocalDefense/Trade channels

### ISSUE THREAD

**Issue by RandomXD** [dalaranwow/dalaran-wow#5940]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
Since early this year or even last year (i dont remember exactly) you cant write on general/localdefense/trade channels anymore, the channels still appear on ur chat channels tab and on the chat whenever u switch zones/log in but when you try to write something on them no message is sent
![Screenshot_1](https://user-images.githubusercontent.com/36212275/103087231-75282e80-45ef-11eb-8aa8-324c05d52070.png)

**Expected behaviour**: Tell us what should happen instead
You should be able to write in these channels at any time
**Steps to reproduce the problem**:

1. go in any zone
2. do /1 for general or /2 for localdefense
3. write anything and see how it doesnt send any message

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Lothloryen** [comment]
Hello, thanks for the report, a fix is coming for this. Cheers!
